corehq/apps/case_importer/templates/case_importer/partials/ko_import_history.html

```diff
@@ -40,13 +40,28 @@
             </div>
             <!--/ko-->
         </td>
+        <td class="col-md-1" data-bind="text: created"></td>
         <td class="col-md-1" data-bind="text: case_type">
         </td>
-        <td class="col-md-1" data-bind="text: user_name">
-        </td>
-        <td class="col-md-5" data-bind="with: task_status">
+        <td class="col-md-4" data-bind="with: task_status">
             {% include 'case_importer/partials/ko_import_status.html' %}
         </td>
+        <td class="col-md-3">
+            <strong data-bind="text: user_name"></strong>
+            <!--ko if: upload_comment_edit_allowed-->
+            <inline-edit params="
+                value: comment,
+                url: hqImport('hqwebapp/js/urllib.js').reverse('case_importer_update_upload_comment', upload_id),
```

Comment:
This is pretty cool - we've got `inline-edit`, `hqImport`, and `registerurl` all working in concert!